src/widget/scroll.rs

```diff
@@ -25,6 +25,23 @@ use crate::piet::RenderContext;
 
 use crate::kurbo::Affine;
 
+#[derive(Debug, Clone)]
+enum ScrollDirection {
+    Horizontal,
+    Vertical,
+    All,
+}
+
+impl ScrollDirection {
+    pub fn max_size(&self, bc: &BoxConstraints) -> Size {
```

Comment:
- does this need to be `pub`?
- A short doc comment would be good here.

Reply:
I don't really know what are the best practices for methods on a private struct. My intent was to emphasize that this method will not be used within the struct but from the outside.

I will add docs.

Reply:
Added docs